src/contracts/middleware/example/ECDSARegistry.sol

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "../RegistryBase.sol";
+
+/**
+ * @title A Registry-type contract using aggregate BLS signatures.
+ * @author Layr Labs, Inc.
+ * @notice This contract is used for
+ * - registering new operators
+ * - committing to and finalizing de-registration as an operator
+ * - updating the stakes of the operator
+ */
+contract ECDSARegistry is RegistryBase {
+    using BytesLib for bytes;
+
+    /// @notice the address that can whitelist people
+    address public whitelister;
+    /// @notice toggle of whether the operator whitelist is on or off 
+    bool public whitelistEnabled;
+    /// @notice operator => are they whitelisted (can they register with the middleware)
+    mapping(address => bool) public whitelisted;
+
+    // EVENTS
+    /**
+     * @notice Emitted upon the registration of a new operator for the middleware
+     * @param operator Address of the new operator
+     * @param socket The ip:port of the operator
+     */
+    event Registration(
+        address indexed operator,
+        string socket
+    );
+
+    /// @notice Emitted when the `whitelister` role is transferred.
+    event WhitelisterTransferred(address previousAddress, address newAddress);
+
+    /// @notice Modifier that restricts a function to only be callable by the `whitelister` role.
+    modifier onlyWhitelister {
+        require(whitelister == msg.sender, "BLSRegistry.onlyWhitelister: not whitelister");
+        _;
+    }
+
+    constructor(
+        IStrategyManager _strategyManager,
+        IServiceManager _serviceManager
+    )
+        RegistryBase(
+            _strategyManager,
+            _serviceManager,
+            1
```

Comment:
note: this is where we are setting the number of quorums to `1`

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/7/commits/68f80a0c541b8ba06d37d8840a96fe9e4beee6e1